pkgs/servers/irrd/default.nix

```
@@ -0,0 +1,40 @@
+{ poetry2nix, pkgs, lib }:
+
+let
+  pythonPackages = (poetry2nix.mkPoetryPackages {
+    projectDir = ./.;
+    overrides = [
+      poetry2nix.defaultPoetryOverrides
+      (import ./poetry-git-overlay.nix { inherit pkgs; })
+      (self: super: {
+
+        irrd = super.irrd.overridePythonAttrs (old: {
+          prePatch = ''
+            sed -i '/wheel/d' setup.py
+          '';
+
+          meta = old.meta // {
+            description = "Internet Routing Registry daemon version 4 is an IRR database server, processing IRR objects in the RPSL format.";
```

```suggestion
            description = "Internet Routing Registry database server, processing IRR objects in the RPSL format";
```
Please read the contributing guidelines.